makeservices/easywp

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Creates symlink to user's webroot. If the symlink exists and files
+# are present in webroot, they are backed up in the user's home directory.
+# A user's database is created or, if it already exists, the password 
+# is reset. WordPress is installed and the wp-config file is filled
+# out with the requisite information. Further configuration can be 
+# done by the user from a browser.  
+#
+
+set -euo pipefail
+shopt -s dotglob
+
+echo -e "\033[31mWARNING: This script should generally be run on a fresh account!\033[00m"
+echo -e "A user with an existing website or database will have their files 
+backed up, but their database password will change and the WordPress 
+install may clutter or try to migrate tables.\n"
+
+read -rp "Are you sure you want to continue? [Y/n] " idunno
```

Comment:
Nit: `n` should be capitalized since it's the default, not `y`.

Reply:
On line 21 the regex ```^$``` will match someone just hitting enter (default) and there is a negation on the if-statement. Or are you asking me to make no the default option?

Reply:
Oh, I wasn't paying attention. Yes, make 'no' the default.